common/actions/rates/actionPayloads.ts

```diff
@@ -1,31 +1,52 @@
 import { handleJSONResponse } from 'api/utils';
 
 export const rateSymbols = ['USD', 'EUR', 'GBP', 'BTC', 'CHF', 'REP', 'ETH'];
-const rateSymbolsArg = rateSymbols.join(',');
 // TODO - internationalize
 const ERROR_MESSAGE = 'Could not fetch rate data.';
 const CCApi = 'https://min-api.cryptocompare.com';
 
-const CCRates = (symbol: string) => {
-  return `${CCApi}/data/price?fsym=${symbol}&tsyms=${rateSymbolsArg}`;
+const CCRates = (symbols: string[]) => {
+  const tsyms = rateSymbols.concat(symbols).join(',');
+  return `${CCApi}/data/price?fsym=ETH&tsyms=${tsyms}`;
 };
 
 export interface CCResponse {
-  symbol: string;
-  rates: {
-    BTC: number;
+  [symbol: string]: {
```

Comment:
This should break the rates reducer test recently merged into `develop`.

Reply:
Good catch, fixed!